.github/workflows/full-test.yml

```diff
@@ -118,13 +116,17 @@ jobs:
               echo "Quality metrics changed"
               echo "QUALITYMETRICS_CHANGED=true" >> $GITHUB_OUTPUT
             fi
-            if [[ $file == *"/sorters/"* ]]; then
+            if [[ $file == *"/sorters/"* && $file != *"/sorters/internal/"* && $file != *"/sorters/internal/"* ]]; then
```

Comment:
Should not one the negative conditions be `$file != *"/sorters/external/"`? I think you are repeating the statement here.